libs/app.bundle.js

```diff
@@ -17335,8 +17391,8 @@ module.exports = function(XMPP, eventEmitter)
                     {
                         var audioMuted = startMuted.attr("audio");
                         var videoMuted = startMuted.attr("video");
-                        APP.UI.setInitialMuteFromFocus((audioMuted === "true"),
-                            (videoMuted === "true"));
+                        eventEmitter.emit(XMPPEvents.START_MUTED_FROM_FOCUS,
```

Comment:
With the previous implementation we are getting booleans too. I'm just saying that we should use "audioMuted" and "videoMuted" variables or if we don't want to use them we should remove them because this is the only place where they were used.